src/Sql/dbo/Views/OrganizationView.sql

```diff
@@ -1,6 +1,49 @@
 ﻿CREATE VIEW [dbo].[OrganizationView]
 AS
 SELECT
-    *
+    O.Id,
+    O.Identifier,
+    O.Name,
+    O.BusinessName,
+    O.BusinessAddress1,
+    O.BusinessAddress2,
+    O.BusinessAddress3,
+    O.BusinessCountry,
+    O.BusinessTaxNumber,
+    O.BillingEmail,
+    O.UsePolicies,
+    O.UseSso,
+    O.UseGroups,
+    O.UseDirectory,
+    O.UseEvents,
+    O.Use2fA,
+    O.UseApi,
+    O.UseResetPassword,
+    O.SelfHost,
+    O.Gateway,
+    O.GatewayCustomerId,
+    O.GatewaySubscriptionId,
+    O.ReferenceData,
+    O.Enabled,
+    O.LicenseKey,
+    O.ApiKey,
+    O.PublicKey,
+    O.PrivateKey,
+    O.TwoFactorProviders,
+    O.ExpirationDate,
+    O.CreationDate,
+    O.RevisionDate,
+    O.OwnersNotifiedOfAutoscaling,
+    O.UseKeyConnector,
+    ISNULL(O.MaxAutoscaleSeats, OPM.MaxAutoScaleSeats) As MaxAutoScaleSeats,
```

Comment:
This will first check the organization table, and if the value is null use the OrganizationPasswordManager table. This works fine assuming we also continue to update the Organization table until we are through the transition phase.

Do we want to update `OrganizationPasswordManager_create` and update to update the organization table for now? The alternative is to update the old stored procedures to upsert into the `OrganizationPasswordManager` table but that might be more complex.

Reply:
We can leave the organization create/update procedures alone for now, and from the code we can pass the information to the Organization table as well, would this work?

Reply:
I've updated the OPM_Create to also update the organization table